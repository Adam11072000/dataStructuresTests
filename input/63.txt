init 1973
addImage 4626
addImage 9277
addImage 9277
addImage -7
getHighestScoredLabel 9277 110
addImage 9277
addImage -1
setLabelScore 7869 162 8294 1911
quit