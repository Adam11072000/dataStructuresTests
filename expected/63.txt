init done.
addImage: SUCCESS
addImage: SUCCESS
addImage: FAILURE
addImage: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: FAILURE
addImage: INVALID_INPUT
setLabelScore: FAILURE
quit done.
